src/main/java/com/microsoft/sqlserver/jdbc/Util.java

```diff
@@ -417,8 +417,8 @@ else if (ch == ':')
                         name = SQLServerDriver.getNormalizedPropertyName(name, logger);
                         if (null != name) {
                             if (logger.isLoggable(Level.FINE)) {
-                                if ((false == name.equals(SQLServerDriverStringProperty.USER.toString()))
-                                        && (false == name.equals(SQLServerDriverStringProperty.PASSWORD.toString())))
+                                if ((!name.equals(SQLServerDriverStringProperty.USER.toString()))
```

Comment:
Unnecessary brackets around both `!name.equals(SQLServerDriverStringProperty.USER.toString()` and `!name.equals(SQLServerDriverStringProperty.PASSWORD.toString()`